### PR TITLE
Created a menu and moved New game & Quit game to it

### PR DESCRIPTION
**Pull request by ThePhilgrim**
Removed sidebar buttons, changed y padding of sliders, changed width & height slider min value to 10

### CODE REVIEW

File: minesweeper.py [hunk 1]
```diff
@@ -285,6 +285,19 @@ def new_game():
...
+# Creates a top menu
+top_menu = tkinter.Menu(root)
+root.config(menu=top_menu)
+
+
+top_menu_game = tkinter.Menu(top_menu)
+if root.tk.call('tk', 'windowingsystem') == 'aqua':
+    top_menu.add_cascade(label="Game", menu=top_menu_game)
+    top_menu_game.add_command(label="New Game        F2", command=new_game)
+    top_menu_game.add_command(label="Quit Game       F10", command=quit_game)
```
Comment (Akuli) [inline minesweeper.py]: Use `accelerator="F2"` instead of spaces

File: minesweeper.py [hunk 2]
```diff
@@ -285,6 +285,19 @@ def new_game():
...
+else:
+    top_menu_game.add_command(label="New Game", command=new_game)
+    top_menu_game.add_command(label="Quit Game", command=quit_game)
```
Comment (Akuli) [inline minesweeper.py]: These should go to `top_menu`, not `top_menu_game`

**Review by Akuli — CHANGES_REQUESTED** [review]

**Comment by ThePhilgrim** [comment]
pushed again

File: minesweeper.py [hunk 3]
```diff
@@ -285,6 +285,20 @@ def new_game():
...
+
+
+top_menu_game = tkinter.Menu(top_menu)
+if root.tk.call('tk', 'windowingsystem') == 'aqua':
+    top_menu.add_cascade(label="Game", menu=top_menu_game)
+    top_menu_game.add_command(label="New Game", accelerator="F2", command=new_game)
+    top_menu_game.add_command(label="Quit Game", accelerator="F10", command=quit_game)
+else:
+    top_menu.add_command(label="New Game", command=new_game)
+    top_menu.add_command(label="Quit Game", command=quit_game)
```
Comment (Akuli) [inline minesweeper.py]: These are missing accelerators

**Review by Akuli — CHANGES_REQUESTED** [review]

**Comment by Akuli** [comment]
I guess you could also make pressing F2 and F10 actually work, instead of just showing them in the menu. Something like this:

```
root.bind('<F2>', new_game)
```

For that to work, `new_game()` must be defined so that it can take an event as an argument. But it's currently called without an event, so we need to make the event optional:

```
def new_game(event=None):
    ...
```

**Comment by Akuli** [comment]
Forgot to mention in review: `# Creates a top menu` comment is unnecessary and not helpful, because the code after it is literally `top_menu = ...`

**Comment by ThePhilgrim** [comment]
> I guess you could also make pressing F2 and F10 actually work, instead of just showing them in the menu. Something like this:
> 
> ```
> root.bind('<F2>', new_game)
> ```
> 
> For that to work, `new_game()` must be defined so that it can take an event as an argument. But it's currently called without an event, so we need to make the event optional:
> 
> ```
> def new_game(event=None):
>     ...
> ```

I was working on this before going to bed yesterday, but could not find how to bind keys. I thought I would fix it today

**Review by Akuli — APPROVED** [review]

**Pull request merged**